tests/unit/s3middleware/objectRestore.spec.js

```diff
@@ -0,0 +1,146 @@
+const assert = require('assert');
+const { convertToXml, parseRestoreRequestXml } =
+    require('../../../lib/s3middleware/objectRestore');
+const DummyRequestLogger = require('../helpers').DummyRequestLogger;
+
+const log = new DummyRequestLogger();
+
+const generateXml = (days, tier) => {
+    const daysElement = days ? `<Days>${days}</Days>` : '';
+    const tierElement = tier ? `<Tier>${tier}</Tier>` : '';
+    return [
+        '<RestoreRequest xmlns="http://s3.amazonaws.com/doc/2006-03-01/">',
+        `${daysElement}`,
+        `${tierElement}`,
+        '</RestoreRequest>',
+    ].join('');
```

Comment:
```suggestion
    const ret = ['<RestoreRequest xmlns="http://s3.amazonaws.com/doc/2006-03-01/">'];
    if (days) {
        ret.push(`<Days>${days}</Days>`);
    }
    if (tier) {
        ret.push(`<Tier>${tier}</Tier>`);
    }
    ret.push('</RestoreRequest>');
    return ret.join('');
```

This style is a bit more flexible.

Reply:
thank you! this looks clearer